docs/features/series.md

```diff
@@ -119,23 +116,23 @@ GET playlist\<playlistid>
      "medaid":"dwEE1oBP",
      "title":"Video Title",
      "description":"Lorem ipsum",
-     "seriesId": "aSZZ1oBP",
+     "contentType": "series",
```

Comment:
Doesn't have to be part of this PR but we should probably add a doc file just to discuss content types and screen mapping.

Reply:
Here I just wanted to show the way a media item could look when creating series on the dashboard page (first phase). In the future, as discussed, different content types can be used for series (`is_series` flag).

Reply:
Yes, though keep in mind that the is_series flag will be on the content schema, not the custom params of the media item.

Reply:
Left TODO and added a small note about another content type.